contracts/auction/LANDAuctionStorage.sol

```diff
@@ -50,6 +50,7 @@ contract LANDAuctionStorage {
     ITokenConverter public dex;
     mapping (address => Token) public tokensAllowed;
     uint256 public totalManaBurned = 0;
+    uint256 public landsBidded = 0;
```

Comment:
I'm ok with the PR. My only comment is maybe a better name is totalLandBidded, like the one for Mana. Feel free to merge.